sympy/physics/wigner.py

```diff
@@ -194,12 +194,28 @@ def wigner_3j(j_1, j_2, j_3, m_1, m_2, m_3):
 
     - Jens Rasch (2009-03-24): initial version
     """
-    if int(j_1 * 2) != j_1 * 2 or int(j_2 * 2) != j_2 * 2 or \
-            int(j_3 * 2) != j_3 * 2:
+
+    def convert_float_to_rational_if_half_integer(value):
+        if isinstance(value, float):
+            fraction_value = Rational(value).limit_denominator()
+            if fraction_value.denominator == 2:
+                return fraction_value
+        return value
```

Comment:
```suggestion
        if isinstance(value, float):
            if equal_valued((v:=2*value), (i:=int(v))):
                return Rational(i, 2)
        return value
```
`equal_valued` can be imported from sympy.core.numbers. Should this be passing if someone passes 3/3?

Reply:
Yes it should pass because it's an integer

Reply:
We don't want to use `limit_denominator` for this.

Reply:
In my machine it's not failing the code quality but on git it's failing it for `equal_valued` why ?